generate-site

```diff
@@ -1,12 +0,0 @@
-#!/bin/sh
-jekyll build
-
-# The following directories are excluded from the list
-# of files to copy on the jekyll build. We use symlinks
-# to them from the _site subdirectory.
-
-# 1. Jekyll doesn't handle ok copying a directory with
-# symlinks (references to the stable/unstable versions)
-ln -s ../reference _site/reference
```

Comment:
Seems the link to /reference is missing?

Reply:
This script was copied from the WebKitGTK website repository. For the moment we do not have reference documentation for the WPE port.

At the moment there is some testing documentation under `/releases` only, but nothing links to it from the generated web site content. My idea was to add a note about `/reference` later on, when we have documentation links in the generated pages... but we can also add it now, if you would prefer that.

Reply:
I see..
Since I saw some folders inside reference I assumed there was some doc, but looking further it looks indeed like a kind of alpha test that was done at some point since there is only for 2.23.90